packages/app/package.json

```diff
@@ -39,6 +39,7 @@
     "@backstage/plugin-techdocs": "^0.9.7",
     "@backstage/plugin-todo": "^0.1.3",
     "@backstage/plugin-user-settings": "^0.2.12",
+    "@backstage/plugin-xcmetrics": "^0.1.1",
```

Comment:
Also here, maybe we don't need to update the example app in the repo.

Reply:
Copy that! 🙂